CRM/Event/Form/Registration.php

```diff
@@ -1598,15 +1598,18 @@ protected function validatePriceSet(array $params, $priceSetId, $priceSetDetails
    * @return mixed|null
    */
   public function getSubmittedValue(string $fieldName) {
-    $value = parent::getSubmittedValue($fieldName);
```

Comment:
As we're explicitly handling Register, Confirm and ThankYou forms via this function we should only call the parent function for any other forms that might come through here.